simulation_parameters/Constants.cs

```diff
@@ -774,6 +775,14 @@ public static class Constants
         "user://mods",
     };
 
+    /// <summary>
+    ///   Regex expresions to categorize different file types.
+    /// </summary>
```

Comment:
This should be a normal comment as this is related to 3 variables and not just this one.